cmd/samToFa/samToFa.go

```diff
@@ -170,9 +177,9 @@ type voteBase struct {
 
 func usage() {
 	fmt.Print(
-		"samToFa - Generate a fasta file from a sam over a refrence sequence. Uncovered sequences are converted to lowercase reference sequences.\n" +
+		"samToFa - Generate a fasta file and accompanying vcf from a sam over a refrence sequence. Uncovered sequences are converted to lowercase reference sequences.\n" +
```

Comment:
A little more info might be good here.  I see that our test file has some insertions in the read mapping, but I think this is only doing SNPs?  I could image a "samToFa" that extracts the read sequences out of a sam file, but this is really a mutation caller.  Maybe samToSnps or samToVcf would be a better name?